ui/src/checks/actions/thunks.ts

```diff
@@ -115,9 +115,15 @@ export const getCheckForTimeMachine = (checkID: string) => async (
 
     const check = resp.data
 
-    const view = createView<CheckViewProperties>(check.type)
+    const normCheck = normalize<Check, CheckEntities, string>(
+      resp.data,
+      checkSchema
+    )
+    const builderCheck = normCheck.entities.checks[normCheck.result]
+
+    const view = createView<CheckViewProperties>(builderCheck.type)
 
-    view.properties.queries = [check.query]
+    view.properties.queries = [builderCheck.query]
```

Comment:
iirc, this is the crux of the issue. we normalized the check, but didn't use the normalized result.